docs/table/access_table.rst

```diff
@@ -726,18 +745,18 @@ expressions (see the warning below for caveats to this)::
 Bytestring columns
 ^^^^^^^^^^^^^^^^^^
 
-Using bytestring columns (numpy ``'S'`` dtype) is straightforward
-with astropy tables since they can be easily compared with the natural
-Python string (``str``) type.  See `The bytes/str dichotomy in Python 3
+Using bytestring columns (``numpy`` ``'S'`` dtype) is possible
+with ``astropy`` tables since they can be compared with the natural
+Python string (``str``) type. See `The bytes/str dichotomy in Python 3
 <https://eli.thegreenplace.net/2012/01/30/the-bytesstr-dichotomy-in-python-3>`_
 for a very brief overview of the difference.
 
-The standard method of representing strings in `numpy` is via the
-unicode ``'U'`` dtype.  The problem is that this requires 4 bytes per
+The standard method of representing strings in `NumPy` is via the
```

Comment:
This is about `numpy` code specifically the way strings are stored in `numpy` arrays.